### ISSUE TITLE
add in support capability/resource and survey synergy into the graph

### ISSUE THREAD

**Issue by sazabi4** [lsstdarkmatter/dark-matter-graph#40]
It would be good to have some information on additional observations/datasets in addition to the LSST data,  e.g. what additional observation/dataset is needed, what telescope/instrument (or ToO) capability is needed, synergies with other telescope or surveys, etc. This will help organize the synergy of LSST with other resources and also give bits of advice on the upcoming 2020 decadal survey.

**Comment by kadrlica** [comment]
Good idea. Do you have an idea of how finely you want to subdivide this? For example, is it enough to say "spectroscopy", do you want to sub-define to "medium resolution spectroscopy" or "wide area spectroscopy", or do you want to put specific instruments?

For context, we currently have a `followup` element that we are using to sporadically track requirements external to LSST. You can see an example [here](https://github.com/lsstdarkmatter/dark-matter-graph/blob/eed8b0269e9b45e0d618bd0ff8f1bc87a833e868/data/data.yaml#L134).

**Comment by sazabi4** [comment]
I think if we are having it as a bullet for connecting different columns, "spectroscopy" is fine, but in the description (or say the white paper), it should be concrete and specific. Otherwise, it will not be informative enough for the decision maker.

For example, spectroscopy needs to specify whether it's single object or multi-object or IFU, wide field or not, high resolution or medium resolution; can we use data from existing or upcoming/planned surveys and instruments or new telescopes/instruments. (Also maybe consider adding in related surveys or related instruments in the description)

For example, proper motion should mention reaching what precision at what depth. Whether GAIA and LSST are enough, or space telescope is needed, etc.

Maybe it's better to have the description as detail as possible for all "measurements" for now, and we can later on decide how many categories we need.

Again, the suggestion is that, if we are going to start with this (awesome) graph for the white paper, this graph should provide a general structure for what to be included in the white paper. And when we divide different measurements into different groups/people for writing, we need to keep in mind ask them to add in the capability/resource/synergy part. This is the motivation of this issue.

Ting

**Comment by kadrlica** [comment]
This sounds like a good plan @sazabi4. We can start with the "Complementary Instruments" column in [table.md](https://github.com/LSSTDESC/LSSTDarkMatter/blob/master/table.md) and expand on that. It would probably be good to have both a "category-level" descriptor (i.e, "spectroscopy", "x-ray", "gamma-ray", "submillimeter") and a more detailed text-level descriptor (i.e., multi-object spectroscopy on an 8m+ instrument, high resolution x-ray calorimetry, etc.).
